packages/dmn-js-decision-table/src/features/decision-table-head/editor/components/InputCellContextMenu.js

```diff
@@ -83,6 +83,8 @@ export default class InputCellContextMenu extends Component {
     return (
       <div className="context-menu-container input-edit">
         <InputEditor
+          defaultExpressionLanguage={ this.getValue('defaultInputExpressionLanguage') }
+          expressionLanguageOptions={ this.getValue('inputExpressionLanguageOptions') }
```

Comment:
This is one of the places where I struggle to understand how to pass those values through from the top-level modeler component.